apps/android/app/src/main/java/chat/simplex/app/model/ChatModel.kt

```diff
@@ -729,7 +731,7 @@ class CIQuote (
   override val text: String get() = content.text
 
   fun sender(user: User): String? = when (chatDir) {
-    is CIDirection.DirectSnd -> "you"
+    is CIDirection.DirectSnd -> generalGetString(R.string.sender_you_pronoun)
```

Comment:
should we change this to show user profile's display name? It was okay in English, but it'll look awkward in Russian quotes